### PR TITLE
Adds Test for TRACEMap

### LINKED ISSUE #1891: Test coverage for XRT and TRACE
The map sources for XRT and TRACE need to have tests.

### PR DESCRIPTION
**Pull request by yashkgp**
Fixes #1891

### CODE REVIEW

File: sunpy/map/sources/tests/test_trace_source.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+def test_is_datasource_for():
+    """Test the is_datasource_for method of TRACEMap.
+    Note that header data to be provided as an argument
+    can be a MetaDict object."""
+    assert trace.is_datasource_for(trace.data, trace.meta)
+
+def test_measurement():
+    """Tests the measurement property of the TRACEMap object."""
+    assert int(trace.measurement) == 171
```
Comment (Cadair) [inline sunpy/map/sources/tests/test_trace_source.py]: does measurement not return a Quantity object? If I remember correctly it should. Is waveunit not set correctly for TRACE?

File: sunpy/map/sources/tests/test_trace_source.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+
+from sunpy.map.sources.trace import TRACEMap
+from sunpy.map import Map
+import sunpy.data.test
+
+path = sunpy.data.test.rootdir
+fitspath = glob.glob(os.path.join(path, "tsi20010130_025823_a2.fits"))
+trace = Map(fitspath)
```
Comment (Cadair) [inline sunpy/map/sources/tests/test_trace_source.py]: this should be a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html)

File: sunpy/map/sources/tests/test_trace_source.py (hunk 1 above)
Comment (yashkgp) [reply sunpy/map/sources/tests/test_trace_source.py]: No, @Cadair waveunit is been set to `None`, and that's why measurement is returning a `str`.

**Comment by Cadair** [comment]
also @yashkgp can you merge the sunpy master branch into this?

File: sunpy/map/sources/tests/test_trace_source.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import glob
+
+import pytest
+
+from sunpy.map.sources.trace import TRACEMap
+from sunpy.map import Map
+import sunpy.data.test
+
+path = sunpy.data.test.rootdir
+fitspath = glob.glob(os.path.join(path, "tsi20010130_025823_a2.fits"))
```
Comment (dpshelio) [inline sunpy/map/sources/tests/test_trace_source.py]: Why do we need to use `glob` in here?

Comment (yashkgp) [reply sunpy/map/sources/tests/test_trace_source.py]: No . I think I forgot to remove it , while experimenting with the code .

File: sunpy/map/sources/tests/test_trace_source.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import os
+import glob
```
Comment (dpshelio) [inline sunpy/map/sources/tests/test_trace_source.py]: You forgot to remove the import

Comment (yashkgp) [reply sunpy/map/sources/tests/test_trace_source.py]: Sorry , My bad .

**Pull request merged**